Examples/Tests/photon_photon_collisions/analysis_photon_photon_collisions.py

```diff
@@ -0,0 +1,200 @@
+        assert(macro_photonA_weight == macro_photonB_weight == 0.)
+        macro_positron_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,6]
+        macro_positron_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,11]
+        macro_electron_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,5]
+        macro_electron_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,10]
+        assert(macro_electron_number == macro_positron_number == 2.)
+        assert(macro_electron_weight == macro_positron_weight == w1 == w2)
+
+def check_energy_conservation():
+    ekin_data = np.loadtxt('diags/reducedfiles/ParticleEnergy.txt')
+    ekin_photonA = ekin_data[:,3]
+    ekin_photonB = ekin_data[:,4]
+    ekin_electron = ekin_data[:,5]
+    ekin_positron = ekin_data[:,6]
+    num_data = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')
+    num_phys_electron = num_data[:,10]
+    num_phys_positron = num_data[:,11]
+    field_energy = np.loadtxt('diags/reducedfiles/FieldEnergy.txt')[:,2]
+    total_energy = ekin_photonA + ekin_photonB + ekin_electron + ekin_positron + m_e*c**2*(num_phys_electron + num_phys_positron) + field_energy
+    assert(np.all(is_close(total_energy, total_energy[0], rtol=5e-12)))
+
+def check_momentum_conservation():
+    total_momentum_x = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,2]
+    total_momentum_y = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,3]
+    total_momentum_z = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,4]
+    assert(np.all(is_close(total_momentum_x, total_momentum_x[0])))
+    assert(np.all(is_close(total_momentum_y, total_momentum_y[0])))
+    assert(np.all(is_close(total_momentum_z, total_momentum_z[0])))
+
+def check_charge_conservation():
+    rho_max = np.loadtxt('diags/reducedfiles/RhoMaximum.txt')[:,2]
+    assert(is_close(rho_max, rho_max[0]).all())
+    #series = io.Series("diags/fields_particles/openpmd_%T.bp",io.Access.read_only)
+    #iterations = np.asarray(series.iterations)
+    #start = series.iterations[iterations[0]]
+    #rho_start = start.meshes["rho"][io.Mesh_Record_Component.SCALAR].load_chunk()
+    #end = series.iterations[iterations[-1]]
+    #rho_end = end.meshes["rho"][io.Mesh_Record_Component.SCALAR].load_chunk()
+    #series.flush()
+
+    filename_end = sys.argv[1]
+    filename_start = filename_end[:-4] + '0000'
+    ds_end = yt.load(filename_end)
+    ds_start = yt.load(filename_start)
+    field_data_end = ds_end.covering_grid(level=0, left_edge=ds_end.domain_left_edge,
+                                          dims=ds_end.domain_dimensions)
+    field_data_start = ds_start.covering_grid(level=0, left_edge=ds_start.domain_left_edge,
+                                              dims=ds_start.domain_dimensions)
+    rho_start = field_data_start["rho"].to_ndarray()
+    rho_end = field_data_end["rho"].to_ndarray()
+    assert(np.all(is_close(rho_start, rho_end)))
```

Comment:
Related to the comment above, could this be simply replaced by the following?
```suggestion
    assert(np.all(np.isclose(rho_start, rho_end, rtol=default_tol, atol=0.)))
```